localtime_test.go

```diff
@@ -293,9 +293,9 @@ func TestDateTimeToString(t *testing.T) {
 		dateTime  LocalDateTime
 		roundTrip bool // ParseLocalDateTime(str).String() == str?
 	}{
-		{"2016-03-22T13:26:33", LocalDateTime{LocalDate{2016, 03, 22}, LocalTime{13, 26, 33, 0}}, true},
-		{"2016-03-22T13:26:33.000000600", LocalDateTime{LocalDate{2016, 03, 22}, LocalTime{13, 26, 33, 600}}, true},
-		{"2016-03-22t13:26:33", LocalDateTime{LocalDate{2016, 03, 22}, LocalTime{13, 26, 33, 0}}, false},
+		{"2016-03-22T13:26:33", LocalDateTime{LocalDate{2016, 0o3, 22}, LocalTime{13, 26, 33, 0}}, true},
```

Comment:
can we just put `3`? an octal literal seems weird :D

Reply:
yep, not sure how this came up... it seems to be gofumpt, but not sure. Luckily it was caught by you.

Reply:
In Go when a literal number is prefixed by `0` it implies the octal notation. I guess whatever tool fixed that decided to make it explicit